script/install.sh

```diff
@@ -82,7 +82,7 @@ retry() {
     for i in {0..4}; do
         if "$@"; then
             break
-        elif [ $i -lt 4 ]; then
+        elif [ "$i" -lt 4 ]; then
```

Comment:
i isn't a string, but a number. No splitting can happen here

Reply:
Yeah, I've done some batch changes and didn't noticed that this was actually useless. If you prefer, I can do a separate PR for `install.sh` scripts.

Reply:
Done